lib/src/model/recipe_stub.dart

```diff
@@ -160,9 +160,8 @@ class _$RecipeStubSerializer implements PrimitiveSerializer<RecipeStub> {
           final valueDes = serializers
               .deserialize(
                 value,
-                specifiedType: const FullType(int),
-              )
-              .toString();
+                specifiedType: const FullType(String),
```

Comment:
Can you try just ommiting the type and leave the `.toString()`?

this should work in both cases for now.